src/server.ts

```diff
@@ -1,14 +1,19 @@
 import { createServer as createServerHTTP } from 'node:http'
 import type { AddressInfo } from 'node:net'
 import type { Options, ServerType } from './types'
-import { getRequestListener } from './listener'
+import { getRequestListener, getUpgradeListener } from './listener'
 
 export const createAdaptorServer = (options: Options): ServerType => {
   const fetchCallback = options.fetch
+  const upgrade = options.upgrade !== false
```

Comment:
This would always add the upgrade listener unless it is actively set to false. Is this intended?

Reply:
Yes, idk if it should be the default or not so i did